index.html

```diff
@@ -3680,11 +3680,57 @@ <h1>Speakers</h1>
                                  <span>Oracle</span>
                              </div>
                          </div>
 
+                          <div class="col-md-3 col-sm-6 speaker-column">
+                             <div class="speaker">
+                                 <div class="image-holder">
+                                     <img class="background-image" alt="Neha Kumari" src="img/NehaKumari.jpg">
+                                     <div class="hover-state text-center preserve3d">
+                                         <div class="social-links vertical-align">
+                                            <a target="default" href="https://oracle.com/"><i class="icon fa fa-external-link"></i></a>
+                                         </div>
+                                     </div>
+                                 </div>
+                                 <span class="speaker-name">Neha Kumari</span>
+                                 <span>Software Developer</span>
+                                 <span>Oracle</span>
+                             </div>
+                         </div>
+
+                          <div class="col-md-3 col-sm-6 speaker-column">
+                             <div class="speaker">
+                                 <div class="image-holder">
+                                     <img class="background-image" alt="Joyce Ng" src="img/JoyceNg.jpg">
+                                     <div class="hover-state text-center preserve3d">
+                                         <div class="social-links vertical-align">
+                                            <a target="default" href="https://9v1anradio.no-ip.info/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://github.com/geekyjoyce72 "><i class="icon fa fa-github"></i></a>
+                                         </div>
+                                     </div>
+                                 </div>
+                                 <span class="speaker-name">Joyce Ng</span>
+                                 <span>Founder</span>
+                                 <span>Hyan Technologies</span>
+                             </div>
+                         </div>
+
+                         <div class="col-md-3 col-sm-6 speaker-column">
+                             <div class="speaker">
+                                 <div class="image-holder">
+                                     <img class="background-image" alt="Ethical Hacking" src="img/EthicalHacking.jpg">
+                                     <div class="hover-state text-center preserve3d">
+                                         <div class="social-links vertical-align">
+                                         </div>
+                                     </div>
+                                 </div>
+                                 <span class="speaker-name">Ethical Hacking</span>
```

Comment:
Provide preview link it seems this may lead to improper alignment.

Reply:
No, I have checked!

Reply:
hi @gautamjajoo , sorry now we have to replace the photo of Ethical Hacking group. I'm asking them to provide a photo with a representative person only. Their group photo will not make other speakers happy...